integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/massindexing/MassIndexingErrorDefaultBackgroundFailureHandlerIT.java

```diff
@@ -52,25 +51,19 @@ protected void assertEntityIndexingFailureHandling(String entityName, String ent
 	}
 
 	@Override
-	protected void expectEntityGetterFailureHandling(String entityName, String entityReferenceAsString,
-			String exceptionMessage, String failingOperationAsString) {
+	protected void expectEntityGetterFailureHandling(String exceptionMessage, String failingOperationAsString) {
 		logged.expectEvent(
 				Level.ERROR,
-				ExceptionMatcherBuilder.isException( SearchException.class )
-						.withMessage( "Exception while invoking" )
-						.causedBy( SimulatedError.class )
+				ExceptionMatcherBuilder.isException( SimulatedError.class )
 						.withMessage( exceptionMessage )
 						.build(),
-				failingOperationAsString,
-				"Entities that could not be indexed correctly:",
-				entityReferenceAsString
+				failingOperationAsString
 		)
-				.once();
+				.times( 2 );
```

Comment:
I wonder why the error is reported twice? There may be a problem here. I'll investigate.